es.lproj/Localizable.strings

```diff
@@ -0,0 +1,187 @@
+
+/* No comment provided by engineer. */
+"Device motion is active" = "Movimiento del dispositivo está activo";
+
+/* No comment provided by engineer. */
+"Device motion is already active." = "Movimiento del dispositivo ya está activo.";
+
+/* No comment provided by engineer. */
+"Dismiss" = "Descartar";
+
+/* No comment provided by engineer. */
+"Get Started" = "Empezar";
+
+/* No comment provided by engineer. */
+"Go Pro" = "Valla pro";
+
+/* No comment provided by engineer. */
+"Good" = "Buena";
+
+/* No comment provided by engineer. */
+"Great" = "Estupenda";
+
+/* No comment provided by engineer. */
+"Loading" = "Cargando";
+
+/* No comment provided by engineer. */
+"Make your phone vibrate whenever your posture is bad." = "Hacer que su teléfono vibre cuando su postura sea mala.";
+
+/* No comment provided by engineer. */
+"More info" = "Más información";
+
+/* No comment provided by engineer. */
+"No pages for this support item" = "No hay páginas para este elemento de apoyo";
+
+/* No comment provided by engineer. */
+"No permission" = "Sin permiso";
+
+/* No comment provided by engineer. */
+"OK" = "OK";
+
+/* No comment provided by engineer. */
+"Play a subtle sound through your AirPods whenever your posture is bad" = "Reproducir un sonido sutil a través de sus AirPods cuando su postura sea mala";
+
+/* No comment provided by engineer. */
+"Please, re-connect your headphones." = "Por favor, vuelva a conectar sus audífonos.";
+
+/* No comment provided by engineer. */
+"Posture\nPal Pro" = "Posture\nPal Pro";
+
+/* No comment provided by engineer. */
+"Posture Pal\nPro" = "Posture Pal\nPro";
+
+/* No comment provided by engineer. */
+"Posture Pal helps you improve your head and neck posture with your AirPods!" = "Posture Pal le ayuda a mejorar la postura de su cabeza y cuello con sus AirPods!";
+
+/* No comment provided by engineer. */
+"Posture Tracking Sensitivity" = "Sensibilidad de seguimiento de postura";
+
+/* No comment provided by engineer. */
+"Privacy Policy" = "Póliza de privacidad";
+
+/* No comment provided by engineer. */
+"Reset" = "Reiniciar";
+
+/* No comment provided by engineer. */
+"Restore Purchases" = "Restaurar compras";
+
+/* No comment provided by engineer. */
+"SCORE" = "Puntos";
+
+/* No comment provided by engineer. */
+"Sensitivity" = "Sensibilidad";
+
+/* No comment provided by engineer. */
+"Sensitivity: %@" = "Sensibilidad: %@";
+
+/* No comment provided by engineer. */
+"Session Complete" = "Sesión completada";
+
+/* No comment provided by engineer. */
+"Settings" = "Configuracion";
+
+/* No comment provided by engineer. */
+"Skip" = "Saltar";
+
+/* No comment provided by engineer. */
+"Sound Alerts Disabled" = "Alertas de Sonido Deshabilitados";
+
+/* No comment provided by engineer. */
+"Sound Alerts Enabled" = "Alertas de Sonido Habilitados";
+
+/* No comment provided by engineer. */
+"Sounds" = "Sonidos";
+
+/* No comment provided by engineer. */
+"Start Tracking" = "Dejar de rastrear";
```

Comment:
Doesn't 'dejar' mean 'stop' not 'start'?

Reply:
Oops yes my mistake, will fix ASAP